lib-extra/build.gradle

```diff
@@ -22,6 +22,5 @@ dependencies {
 }
 
 // we'll hold the core lib to a high standard
-findbugs {
-	reportLevel = 'low' // low|medium|high (low = sensitive to even minor mistakes)
+findbugs { reportLevel = 'low' // low|medium|high (low = sensitive to even minor mistakes)
 }
```

Comment:
I think it is the comment, which is in the same line. The Groovy-Formatter seems only to accept this or method calls.